javascript/javascript1/week3/speed-function.js

```diff
@@ -0,0 +1,13 @@
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+};
+  
+function howlongTakes (travelInformation){ 
+    let hours = travelInformation.destinationDistance / travelInformation.speed
+    let minutes = (hours - Math.floor(hours)) * 60
```

Comment:
These should be `const`.

Reply:
Thanks for all comments. I learned much from them.